code/modules/antagonists/villain/zizocult.dm

```diff
@@ -60,15 +60,13 @@ GLOBAL_LIST_EMPTY(ritualslist)
 
 	if(islesser)
 		add_objective(/datum/objective/zizoserve)
-		owner.current.adjust_skillrank(/datum/skill/combat/knives, 2, TRUE)
-		owner.current.adjust_skillrank(/datum/skill/combat/swords, 2, TRUE)
```

Comment:
use clamped_adjust_skillrank instead of removing it zizo cultists themselves are the only conversion antag that is limited